backend/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get remove -y linux-libc-dev && \
 # Set up application files
 WORKDIR /app
 COPY ./danswer /app/danswer
+COPY ./model_server /app/model_server
```

Comment:
This doesn't feel ideal to have `danswer` directly depend on `model_server` AND have `model_server` directly depend on `danswer`. How do you feel about moving all the models into the `danswer` side of things to make it a one way import?

Reply:
Was considering that as well, seems fine